.github/workflows/validation.yml

```diff
@@ -330,3 +332,32 @@ jobs:
           SKIP_LABEL_NAME: ${{ needs.discover.outputs.label_copyright }}
         run: |
           ./.github/scripts/validation_run.sh ./testing/validate_copyright.sh
+
+  markdown_validation:
+    name: Markdown Linter
+
+    needs:
+      - discover
+      - pull_request_info
+    if: needs.discover.outputs.run_markdown == 'true'
+    runs-on: ubuntu-latest
+    steps:
+
+      # <template: checkout_step>
+      - name: Checkout sources
+        uses: actions/checkout@v2.4.0
+        with:
+          ref: ${{ needs.pull_request_info.outputs.ref }}
+      # </template: checkout_step>
+
+      - name: Restore diff artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: pr_diff
+
+      - name: Run check
+        env:
+          DIFF_PATH: ./pr.diff
+          SKIP_LABEL_NAME: ${{ needs.discover.outputs.label_markdown }}
+        run: |
+          ./.github/scripts/validation_run.sh ./testing/validate_markdown.sh
```

Comment:
1. I think having a dedicated script for CI is excessive.
2. Why do we need to use the `validation_run.sh` here? I thought we agreed to run a markdown linter for every file every time. How long does it take to run the validation for all files?

Reply:
Changes:
- Removed `validation_run.sh` script
- Added workflow template for linters

Linter passes about 3-4 seconds over a whole repo.

Examples:
- [Successful job example](https://github.com/deckhouse/deckhouse-test-1/runs/6971723727?check_suite_focus=true)
- [Failed job example](https://github.com/deckhouse/deckhouse-test-1/runs/6971690079?check_suite_focus=true)

@nabokihms

Reply:
I think yes.